plan/builders.go

```diff
@@ -80,8 +80,8 @@ type Builder interface {
 	CreateTableAsSelect(input Rel, tableName []string, schema types.NamedStruct) (*NamedTableWriteRel, error)
 	CrossRemap(left, right Rel, remap []int32) (*CrossRel, error)
 	Cross(left, right Rel) (*CrossRel, error)
-	FetchRemap(input Rel, offset, count uint64, remap []int32) (*FetchRel, error)
-	Fetch(input Rel, offset, count uint64) (*FetchRel, error)
+	FetchRemap(input Rel, offset, count int64, remap []int32) (*FetchRel, error)
```

Comment:
I suggest we add a FETCH_COUNT_ALL_RECORDS const or similar and add doc on this function and the below mentioning it's use.